pyhoma/client.py

```diff
@@ -96,6 +100,9 @@ async def get_devices(self, refresh: bool = False) -> List[Device]:
 
         return devices
 
+    @backoff.on_exception(
+        backoff.expo, NotAuthenticatedException, max_tries=2, on_backoff=relogin
```

Comment:
In general, should we not reconnect on `DisconnectedServerError` as well?

Reply:
I'm not sure we can get this error now we handle properly the logout.